libyui-rest-api/src/YHttpWidgetsActionHandler.cc

```diff
@@ -391,22 +391,9 @@ int YHttpWidgetsActionHandler::do_action(YWidget *widget, const std::string &act
                 }
             } );
         }
-        else if ( dynamic_cast<YDumbTab*>(widget) )
+        else if ( auto tab = dynamic_cast<YDumbTab*>(widget) )
         {
-            return action_handler<YDumbTab>( widget, body, [&] (YDumbTab *tab) {
-                YItem * item = tab->findItem( value );
-                if ( item )
-                {
-                    yuiMilestone() << "Activating Tree Item \"" << item->label() << '"' << std::endl;
-                    tab->setKeyboardFocus();
-                    tab->selectItem( item );
-                    get_widget_handler()->activate_widget( tab );
-                }
-                else
-                {
-                    throw YUIException("Item: '" + value + "' cannot be found among tabs");
-                }
-            } );
+            return action_handler<YDumbTab>( widget, body, get_dumb_tab_handler()->get_handler(tab, value));
```

Comment:
Let's try not to forget adding spaces before and after arguments in the method calls. Even though I'm aware that it's not strictly followed everywhere.